controller/validation.js

```diff
@@ -0,0 +1,6 @@
+const mongoose = require("mongoose");
+
+exports.isValidObjectId = (id) => {
+  let isValid = mongoose.Types.ObjectId.isValid(id);
+  return isValid;
+};
```

Comment:
This should not be in the controllers directory.
Make a separate directory for validators.

Reply:
updated